test/Microsoft.Health.Dicom.Web.Tests.E2E/Rest/DropInvalidMetadataTests.cs

```diff
@@ -17,7 +17,7 @@
 namespace Microsoft.Health.Dicom.Web.Tests.E2E.Rest;
 
 
-[Trait("Category", "bvt-dp")]
+[Trait("Category", "leniency")]
```

Comment:
I am curious on so many E2E tests? My simple brain was expecting these to be unit tests and 1 E2E test to validate the wiring.

Reply:
I always look at tests has pyramid, more unit tests and 1 per functionality E2E test. This is just a part of STOW operation. You just need 1 E2E test that validates valid data went in, invalid data was reported. Rest should be unit tests